Mathlib/MeasureTheory/Measure/WithDensityFinite.lean

```diff
@@ -53,7 +53,7 @@ noncomputable def Measure.toFiniteAux (μ : Measure α) [SFinite μ] : Measure 
   if IsFiniteMeasure μ then μ else (exists_isFiniteMeasure_absolutelyContinuous μ).choose
 
 /-- A finite measure obtained from an s-finite measure `μ`, such that
-`μ = μ.toFinite.withDensity (μ.rnDeriv µ.toFinite)`
+`μ = μ.toFinite.withDensity (μ.rnDeriv μ.toFinite)`
```

Comment:
I can't see the difference here.

Reply:
Yeah, sorry, this one is subtle. We're swapping a micro sign (U+00B5) with a Greek small letter mu (U+03BC), since the micro sign was deviating from the rest of the file.